pkgs/top-level/python-packages.nix

```diff
@@ -8522,7 +8526,13 @@ in {
 
   webthing = callPackage ../development/python-modules/webthing { };
 
-  werkzeug = callPackage ../development/python-modules/werkzeug { };
+  werkzeug = callPackage ../development/python-modules/werkzeug {
+    pytestCheckHook = self.pytestCheckHook.override { pytest = self.pytest_6_1; };
+    pytest-timeout = self.pytest-timeout.override {
+      pytest = self.pytest_6_1;
+      pytestCheckHook = self.pytestCheckHook.override { pytest = self.pytest_6_1; };
```

Comment:
How about creating `pytest6_1CheckHook`?

Reply:
Yes, or revert the pytest update. We should give packages time to update, although they've had half a year.

Reply:
I don't like reverting stuff so the ``pytest_6_1CheckHook`` it is.